runtime/v2/example/README.md

```diff
@@ -0,0 +1,4 @@
+# Example Shim
+
+This directory provides skeleton code as the starting point for creating a Runtime v2 shim.
+This allows runtime authors to quickly bootstrap new shim implementations.
```

Comment:
Should we include a note about the naming scheme?  I know we have a comment about it but I think that can trip authors up sometimes.

Reply:
yes, i can comment or link to the overall documentation for building shims